### DISCUSSION TITLE
ストック共有機能

### DISCUSSION THREAD

**Discussion by irei-shake** [increments/qiita-discussions#697] [要望・アイデア・不具合]
ストックした記事群をカテゴリ単位で共有リンク、もしくはファイル形式でのエクスポート/インポートなどの形式で共有できると嬉しいです。(YouTubeの再生リスト的な)
「このストック共有しとくから記事読み込んどいてね～」って他者に共有できると引継ぎや教育、SNSなどいろんな場面で便利だと思います。

Qiita Teamではチーム内で同一ストックの閲覧などもできるようですが、社外などもっとカジュアルに使用できればと…

こういった機能含めQiita Teamを利用する付加価値として提供している、であればそれは納得できますし、
そもそも最初から自分で共有したい記事をブラウザのブックマークでまとめとけばいい話でもあるので、もし実装する余裕がある＆一般開放しても問題ないという判断であれば実装いただきたいです。

**Comment by tanaka-asami415** [comment]
@irei-shake 
リクエストありがとうございます！

直近の開発優先度などから着手は先になりますが、同様のご意見（https://github.com/increments/qiita-discussions/discussions/411 ）もいただいており、運営としても実装したいと思っております💪

**Reply by irei-shake** [reply]
確認していたつもりでしたが既出のリクエストだったようで失礼しました。
ありがとうございます、気長に待たせていただきます！

**Reply by tanaka-asami415** [reply]
とんでもないです。類似のリクエストでも、開発優先度を考える際の参考になるので大変ありがたいです！
引き続きよろしくお願いします🙇

**Comment by kimkim0814** [comment]
@irei-shake 
お待たせしました！

以前いただいたリクエストにお応えできる「ストックのシェア機能」をベータ版としてリリースしました🎉
ストックリスト（今回のリリースに伴い、「カテゴリ」から「リスト」へ名称が変更になりました。）単位で簡単にシェアいただけるようになりました！

これから正式リリースに向けて、改善や機能追加も予定しています！

ぜひ新機能をご活用いただき、気づいたことやご意見をフィードバックいただけると嬉しいです✨
[詳細はこちらをご覧ください。](https://blog.qiita.com/stock-share-beta/)

リクエストいただきありがとうございました！
